PROJECT

```diff
@@ -39,6 +39,7 @@ resources:
     webhookVersion: v1
 - api:
     crdVersion: v1
+    namespaced: false
```

Comment:
We don't need this. I have added the annotation to include scope https://github.com/VaishnaviHire/opendatahub-operator/blob/initialize_api/apis/components/v1/dashboard_types.go#L41